src/service/print/getdocxdata.js

```diff
@@ -295,7 +295,8 @@ function create(data, paperData) {
 
 function instructions(data) {
   const arr = [];
-  data
+  if(data !== undefined){
```

Comment:
Please check if you can you lodash function `_.isUndefined` to check if value is `undefined`

Reply:
okay